paket.lock

```diff
@@ -2,15 +2,15 @@ NUGET
   remote: http://nuget.org/api/v2
   specs:
     Expression.Blend.Sdk (1.0.2)
-    FAKE (3.36.0)
-    Fantomas (1.8.0-beta)
-      FSharp.Compiler.Service (>= 1.4.0-beta)
+    FAKE (4.0.3)
+    Fantomas (1.7.0)
```

Comment:
The formatting test can be fixed by reverting to Fantomas 1.8.0-beta.

The other tests fail due to AppVeyor timing changed (they probably moved us to a slow image).